services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -1119,4 +1121,24 @@ module.exports = {
       });
     },
   },
+
+  /**
+   *
+   */
+  Mutation: {
+    contentCompanyUpdate: async (_, { input }, { base4rest, basedb }, info) => {
+      validateRest(base4rest);
```

Comment:
Should require auth

Reply:
`@requiresAuth` is present on the [definition](https://github.com/base-cms/base-cms/pull/278/files#diff-bc933e054c1ded476ecb27848eaf79ffR10)